MovieQuiz/Presentation/MovieQuizPresenter.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  MovieQuizPresenter.swift
+//  MovieQuiz
+//
+//  Created by Александр Сироткин on 05.01.2023.
+//
+
+import Foundation
+import UIKit
+
+final class MovieQuizPresenter : QuestionFactoryDelegate {
+
+    let questionsAmount: Int = 10
+
+    private weak var controller: MovieQuizViewControllerProtocol?
+    private var questionFactory: QuestionFactoryProtocol?
+    private var statisticService: StatisticServiceProtocol?
+
+    private var currentQuestion: QuizQuestion? = nil
+    private var currentQuestionIndex = 0
+    private var correctAnswersPrivate = 0
+
+    init (controller: MovieQuizViewControllerProtocol) {
+        self.controller = controller
+        questionFactory = QuestionFactoryImdb(
+            moviesLoader: MoviesLoader(networkClient: NetworkClient()))
+        questionFactory?.setDelegate(delegate: self)
+        statisticService = StatisticServiceUserDefaults()
+    }
+
+    var correctAnswers: Int {
+        get {
+            correctAnswersPrivate
+        }
+    }
```

Comment:
если используется только get-ер, можно get опускать. 

А вообще, чтобы не делать лишних переменных в swift есть вот такой модификатор доступа `private(set) var` -  внутри класса сможешь мутировать объект, а извне только читать.

```suggestion
    var correctAnswers: Int {
        correctAnswersPrivate
    }
```